README.md

```diff
@@ -305,7 +275,9 @@ Read `yakut monitor --help` for details.
 
 ```bash
 $ export UAVCAN__CAN__IFACE="socketcan:can0 socketcan:can1 socketcan:can2"  # Triply-redundant UAVCAN/CAN
-$ yakut monitor
+$ export UAVCAN__CAN__MTU=8                     # Force MTU = 8 bytes
+$ export UAVCAN__CAN__BITRATE=1000000 1000000   # Disable BRS, use the same bit rate for arbitration/data
```

Comment:
```export UAVCAN__CAN__BITRATE="1000000 1000000"```

I believe quotation marks are required here, yes?

Reply:
Bash/zsh/sh should not require them but PowerShell does

Reply:
Turns out I was wrong, sorry, let me fix it.